README.md

```diff
@@ -1,6 +1,8 @@
 Welcome to the OpenSSL Project
 ==============================
 
+This is sftcd's fork of OpenSSL to implement Hybrid Public Key Encryption (HPKE).
```

Comment:
Probably this isn't appropriate for inclusion in this PR ;-)

Reply:
yeah - I figured I'd be making changes and could fix that along the way:-)